gbpservice/nfp/orchestrator/modules/service_orchestrator.py

```diff
@@ -603,9 +612,16 @@ def create_network_function(self, context, network_function_info):
         return network_function
 
     def update_network_function(self, context, network_function_id,
-                                updated_network_function):
+                                user_config):
         # Handle config update
-        pass
+        user_config = jsonutils.dumps(user_config)
```

Comment:
It seems user_config should be config string here. But in the node driver the update RPC is invoked by passing config=context.current_node